src/frontend/src/test-e2e/flows.ts

```diff
@@ -172,8 +165,8 @@ export const FLOWS = {
     browser: WebdriverIO.Browser,
     recoveryPhrase: string
   ): Promise<void> => {
-    const welcomeView = new WelcomeView(browser);
-    await welcomeView.recover();
+    const authenticateView = new AuthenticateView(browser);
```

Comment:
Why this change?

The user arrives at a different page if the storage is not cleared?

Reply:
Yes, see explanation [here](https://github.com/dfinity/internet-identity/pull/1929#issuecomment-1734928421).